sdk/src/builder.rs

```diff
@@ -249,6 +250,32 @@ impl Builder {
         self
     }
 
+    /// Sets the remote_url for this [`Builder`].
+    /// The url will be injected into the destination asset when signing.
+    /// It is used to a remote location where the manifest can be retrieved.
+    /// # Arguments
+    /// * `url` - The URL where the manifest will be available.
+    /// # Returns
+    /// * A mutable reference to the [`Builder`].
+    pub fn set_remote_url(&mut self, url: &str) -> &mut Self {
+        self.remote_url = Some(url.to_string());
+        self
+    }
+
+    /// Sets the no_embed flag for this [`Builder`].
+    /// If true, the manifest store will not be embedded in the destination asset on sign.
+    /// This is useful for cloud only and sidecar manifests.
+    /// Note that if a remote URL is set, the sign output will be modified to reference the remote URL.
```

Comment:
Yeah I debated about this. Our current practice is to not insert a URL when embedding, since it leads apps like verify to do a remote fetch instead of getting the embedded manifest. But then there is no reference to the remote manifest. I think giving the API the flexibility to do it either way seems ok, but we can comment that when embedding it is better not to include the remote url.

Reply:
I think it's a bit more intuitive for no_embed=true to be implicitly specified if remote_url is specified. However, if no_embed is explicitly set to false, then it will do both. That's the general assumption I had and I wonder if other people feel the same way.

Reply:
The current practice is to allow setting both remote url and embedded manifest. This allows the developer the most flexibility so I'm going to leave it as is with additional comments to let the user know.